docker/Dockerfile

```diff
@@ -13,7 +13,8 @@ ENV PYTHONPATH=/app
 
 # Install system dependencies required for OpenCV
 RUN apt-get update && apt-get install -y --no-install-recommends \
-    libgl1-mesa-glx=22.3.6-1+deb12u1 \
+    libgl1 \
+    libglx-mesa0 \
```

Comment:
This is coming from pre-commit hook:

```bash

Lint Dockerfiles.........................................................Failed
- hook id: hadolint
- exit code: 1

docker/Dockerfile:15 DL3008 warning: Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`
```